responses.py

```diff
@@ -49,6 +50,8 @@ def wrapper%(signature)s:
 
 logger = logging.getLogger('responses')
 
+psl = PublicSuffixList()
```

Comment:
unused

Reply:
Ugh, sorry about these mistakes. I've fixed them up and I'll be more vigilant next time. Thanks